staging/src/k8s.io/apiserver/pkg/util/flag/map_string_bool.go

```diff
@@ -37,6 +37,10 @@ func (m MapStringBool) String() string {
 
 // Set implements github.com/spf13/pflag.Value
 func (m MapStringBool) Set(value string) error {
+	// clear old values
+	for k := range m {
```

Comment:
Should only be done on first call to Set

Reply:
Done